osu.Game.Rulesets.Taiko/Objects/Swell.cs

```diff
@@ -15,5 +20,18 @@ public class Swell : TaikoHitObject, IHasEndTime
         /// The number of hits required to complete the swell successfully.
         /// </summary>
         public int RequiredHits = 10;
+
+        public List<DrumSampleMapping> ProgressionSamples = new List<DrumSampleMapping>();

```

Comment:
what does progression mean? if we're using this term it definitely deserves a comment

Reply:
Samples during the active time of the Swell, got a better name for this?